book/doctrine.rst

```diff
@@ -641,6 +641,11 @@ to easily fetch objects based on multiple conditions::
 
     If you click the icon, the profiler will open, showing you the exact
     queries that were made.
+    
+    If you make more than 50 queries on a page, the icon will turn yellow to
+    let you know that you are making a lot of requests to the database. This
+    is only a warning and does not have any effect on the execution of further
+    queries.
```

Comment:
It is only a warning, but it means there is potentially an issue in your page (N+1 requests because of proxy initialization in a loop for instance). The warning is here for a reason. Such number of queries is often an issue